packages/react/src/Suspensive.spec.tsx

```diff
@@ -13,6 +13,7 @@ const FALLBACK_GLOBAL = 'FALLBACK_GLOBAL'
 describe('<SuspensiveProvider/>', () => {
   it('should provide default ms prop of Delay', async () => {
     render(
+      // eslint-disable-next-line @typescript-eslint/no-deprecated
```

Comment:
suggestion) 
This PR adds this `eslint-disable-next-line` in several locations.
How about having it at 'warn' level so we can pass CI and no need to have so many eslint-disables..?

Reply:
Using eslint-disable-next-line excessively is indeed an issue, but I’m not entirely sure if changing the rule to 'warn' is the right approach. This change would affect other files, which makes me hesitant.

For example, the most commonly disabled rule is @typescript-eslint/no-deprecated. The places where this rule is disabled involve components like Suspensive and DevMode, which are deprecated and not exposed to users. Since these are very specific cases, I’m concerned that changing the rule to 'warn' might unintentionally affect other parts of the codebase where the rule should still be enforced.

Reply:
I think the "best" solution is to actually replace the deprecated syntax with new syntax:

![Captura de pantalla 2024-10-20 a las 1 14 15 p  m](https://github.com/user-attachments/assets/c21f7023-1d32-47eb-954e-e9a1a04d7a01)

Reply:
This file is intended to test components like Suspensive and SuspensiveProvider. Simply replacing them with DefaultPropsProvider, which doesn’t align with the original purpose, doesn’t seem like a good solution.

I believe that these deprecated files will likely be removed when Suspensive is upgraded to V3. Until then, if there isn’t a better alternative, how about temporarily suppressing the errors with eslint-disable-next-line?

Reply:
This thread is getting long, so to speed up the merge of this PR, I would appreciate it if you could first proceed in the way @SEOKKAMONI did, and then raise this issue later. I will create this as a separate issue.

Reply:
Let's continue the discussion further in issue #1331! I’d like to express my gratitude to @saul-atomrigs for leaving a thoughtful comment that helps us dive deeper into this topic!

Reply:
@SEOKKAMONI thank you and great effort in this PR..!